ast/shared/src/main/scala/jawn/ast/JValue.scala

```diff
@@ -81,6 +82,7 @@ sealed abstract class JValue {
 
 object JValue {
   implicit val facade: Facade[JValue] = JawnFacade
+  implicit lazy val classTag: ClassTag[JValue] = ClassTag(classOf[JValue])
```

Comment:
Complete nitpick, but does this need to be public?  Seems like an implementation detail.

Reply:
Totally legit nitpick, I made it private in https://github.com/typelevel/jawn/pull/547/commits/c61b9512fa66e385caeae46d335c506799daa819

I had to add a scope to the private field otherwise the compilation fails with

```
[error] /home/agourlay/Workspace/jawn/ast/shared/src/main/scala/jawn/ast/JawnFacade.scala:56:55: lazy value classTag in object JValue cannot be accessed in object org.typelevel.jawn.ast.JValue
[error]  Access to protected lazy value classTag not permitted because
[error]  enclosing <$anon: org.typelevel.jawn.FContext.NoIndexFContext[org.typelevel.jawn.ast.JValue]> is not a subclass of
[error]  object JValue in package ast where target is defined
[error]       def finish(): JValue = JArray(vs.toArray(JValue.classTag))

```